bin/cron/applab_datasets/daily_weather

```diff
@@ -163,7 +163,8 @@ def get_weather_data
 
     url = "http://api.openweathermap.org/data/2.5/forecast/daily?zip=#{office.zip_code},us&cnt=#{NUM_DAYS}&units=imperial&appid=#{API_KEY}"
     response = Net::HTTP.get_response(URI(url))
-    if response.code != '200'
+    raise "Response #{response} from #{office.city}" if response.code != '200'
```

Comment:
I think you'll need to wrap this in an explicit `begin`, rather than trying to rescue as part of the `each` block.

```ruby
    begin
      response = Net::HTTP.get_response(URI(url))
      raise "Response #{response} from #{office.city}" if response.code != '200'
    rescue
      Honeybadger.notify(
```